google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -380,7 +384,10 @@ def on_publish_done(future):
                 if retry is gapic_v1.method.DEFAULT:
                     # use the default retry for the publish GRPC method as a base
                     transport = self.api._transport
-                    retry = transport._wrapped_methods[transport.publish]._retry
+                    retry = typing.cast(
```

Comment:
We might avoid the `cast` here, too, by using a different variable name, e.g., `base_retry`.